packages/ui/components/form/inputs/HintOrErrors.tsx

```diff
@@ -3,27 +3,38 @@ import { useFormContext } from "react-hook-form";
 
 import { Check, Circle, Info, X } from "../../icon";
 
-export function HintsOrErrors<T extends FieldValues = FieldValues>(props: {
+type hintsOrErrorsProps = {
   hintErrors?: string[];
   fieldName: string;
+  validationErrors?: string[];
   t: (key: string) => string;
-}) {
+};
+
+export function HintsOrErrors<T extends FieldValues = FieldValues>({
+  hintErrors,
+  fieldName,
+  t,
+  validationErrors,
+}: hintsOrErrorsProps) {
   const methods = useFormContext() as ReturnType<typeof useFormContext> | null;
   /* If there's no methods it means we're using these components outside a React Hook Form context */
   if (!methods) return null;
   const { formState } = methods;
-  const { hintErrors, fieldName, t } = props;
 
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
-  const fieldErrors: FieldErrors<T> | undefined = formState.errors[fieldName];
+  const formFieldErrors: FieldErrors<T> | undefined = formState.errors[fieldName];
+
+  let Errors = validationErrors ? validationErrors : [];
+  if (formFieldErrors?.message.length && !Errors.includes(formFieldErrors.message))
+    Errors.push(formFieldErrors.message);
 
-  if (!hintErrors && fieldErrors && !fieldErrors.message) {
-    // no hints passed, field errors exist and they are custom ones
+  // no hints passed, field errors exist and they are custom ones
```

Comment:
Was not able to figure out how this scenario happens.
Some information regarding this will be helpful in inspecting it further.